grai-integrations/source-postgres/tests/test_analysis.py

```diff
@@ -4,10 +4,10 @@
 
 test_credentials = {
     "host": "localhost",
-    "port": "5432",
-    "dbname": "grai",
-    "user": "grai",
-    "password": "grai",
+    "port": "5433",
```

Comment:
I believe these changes will break our postgres tests (at least running out of the box). I'm guessing you had two postgres instances running at the same time, one for jaffle shop and one for the grai-server? Might be easier to switch the default settings on the jaffle shop repo for that scenario rather than changing them here.